src/pyk/kore/rpc.py

```diff
@@ -110,7 +110,8 @@ def request(self, method: str, **params: Any) -> dict[str, Any]:
         _LOGGER.info(f'Sending request to {server_addr}: {old_id} - {method}')
         req = json.dumps(payload)
         if self._bug_report:
-            bug_report_request = f'rpc/{old_id:03}_request.json'
+            id_with_port = f'{self._port}_{old_id:03}'
+            bug_report_request = f'rpc/{id_with_port}_request.json'
```

Comment:
What do you think?

```suggestion
            bug_report_request = f'rpc_{bug_report_id}/{old_id:03}_request.json'
```

where `bug_report_id` is based on `id(self)`, `self._port`, object creation timestamp, or some combination of these (maybe hashed).

Reply:
Maybe it should be the responsibility of `BugReport` to create a unique folder (with an optional prefix) for clients and return it's name.

Reply:
I went with `id(self)` which works correctly and is pretty simple and readable.